syft/frameworks/torch/tensors/interpreters/gradients_core.py

```diff
@@ -38,6 +46,70 @@ def __call__(self, grad):
     def __repr__(self):
         return self.__class__.__name__
 
+    def __setattr__(self, name, value):
+        self.__dict__[name] = value
+
+        # add attributes of grad function to _parameters list
+        # essential for serialization and deserialization
+        if not name in ["next_functions", "result", "_parameters"]:
```

Comment:
Maybe this is not restrictive enough, can you add a check on the type (should be a Tensor I think)

Reply:
So the problem here is there might be a case where attribute is not a Tensor (consider power in PowBackward function). 
Can you explain a case for which this might not be restrictive enough?

Reply:
Oh you're right!
My only concern is that if people add new internal attributes to GradFunc, then they will need to add them to this list. But maybe this is ok for the moment. Just change the list to be a set :)

Reply:
Done :+1: